spec/druid_tools_spec.rb

```diff
@@ -115,25 +117,41 @@
     allow(Deprecation).to receive(:warn)
     expect(File.exist?(tree1)).to be false
     expect(File.exist?(tree2)).to be false
+    expect(File.exist?(tree3)).to be false
 
     druid1 = described_class.new(druid_str, fixture_dir)
     druid2 = described_class.new(strictly_valid_druid_str, fixture_dir)
+    druid3 = described_class.new(access_druid_str, fixture_dir)
 
     druid1.mkdir
     expect(File.exist?(tree1)).to be true
     expect(File.exist?(tree2)).to be false
+    expect(File.exist?(tree3)).to be false
 
     druid2.mkdir
     expect(File.exist?(tree1)).to be true
     expect(File.exist?(tree2)).to be true
+    expect(File.exist?(tree3)).to be false
+
+    druid3.mkdir
+    expect(File.exist?(tree1)).to be true
+    expect(File.exist?(tree2)).to be true
+    expect(File.exist?(tree3)).to be true
+
+    druid3.rmdir
+    expect(File.exist?(tree1)).to be true
+    expect(File.exist?(tree2)).to be true
+    expect(File.exist?(tree3)).to be false
 
     druid2.rmdir
     expect(File.exist?(tree1)).to be true
     expect(File.exist?(tree2)).to be false
+    expect(File.exist?(tree3)).to be false
 
     druid1.rmdir
     expect(File.exist?(tree1)).to be false
     expect(File.exist?(tree2)).to be false
+    expect(File.exist?(tree3)).to be false
```

Comment:
These methods (rmdir, mkdir, prune! below) are deprecated, yes, but these test changes establish that acting on a single druid does not affect other druids.

Reply:
maybe put this comment in the spec code?

Reply:
@ndushay I'll add some words to the `it` string above.